metadata.tex

```diff
@@ -53,6 +54,13 @@
 \usepackage[skip=6pt, indent=0pt]{parskip}
 \setstretch{1}
 
+% Get version from env
+% \getenv{variable_name} just prints the value
+% \getenv[\macro]{variable_name} stores the value in \macro for reusability
+\newcommand{\getenv}[2][]{%
+  \CatchFileEdef{\value}{"|kpsewhich --var-value #2"}{\endlinechar=-1}%
```

Comment:
Sorry, why do you need to run external command (part of kpathsea) for this?

Reply:
Sorry, I'm not latex pro and I didn't find a better way to read env variables

Reply:
Don't worry, me neither :D

Reply:
Let's continue here https://github.com/Heegu-sama/Homm3BG/pull/555/files#r1627353364